test/integration/demo/demo.test.js

```diff
@@ -33,9 +33,9 @@ describe('obt demo', function () {
 					return execa(obt, ['demo']);
 				})
 				.then(() => {
-					return Promise.reject(new Error('obt demo should error when trying to create demos for a component which has no demos.'));
+					throw new Error('obt demo should error when trying to create demos for a component which has no demos.');
```

Comment:
I think in the tests we should use async await (if we can) and then use proclaim.throws to show it errors correctly